### PR TITLE
[stable/prometheus] add required tillerVersion to Chart.yaml

### PR DESCRIPTION
**Pull request by hoesler**
In reaction to #3546, users reported issues with tiller 2.7. This PR adds `tillerVersion: ">=2.8.0"` to Charts.yaml. Additionally, it fixes some typos.

### CODE REVIEW

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -13,3 +13,4 @@ maintainers:
   - name: mgoodness
     email: mgoodness@gmail.com
 engine: gotpl
+tillerVersion: ">2.8.0"
```
Comment (eicnix) [inline stable/prometheus/Chart.yaml]: Shouldn't it be `>=2.8.0` to also allow 2.8.0?

Comment (hoesler) [reply stable/prometheus/Chart.yaml]: Yes, you are right. Thanks for spotting this.

**Comment by mattfarina** [comment]
/hold

The Charts CI system is still using tiller 2.7. We need to upgrade that.

**Comment by instigardo** [comment]
How is this possible?
Chart incompatible with Tiller v2.10.0-rc.1

**Comment by gianrubio** [comment]
@instigardo what's the error?

**Pull request merged**